bin/metadata_validation.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import json
+
+import synapseclient
+from synapseclient import Entity
+
+
+def get_synaspe_file(syn_id):
+    """
+    Initializes synapseclient session and retrieves file metadata
+    """
+    syn = synapseclient.Synapse()
+    syn.login(authToken=os.environ["SYNAPSE_AUTH_TOKEN"])
+    syn_file = syn.get(syn_id)
```

Comment:
If you just want to download the Synapse entity metadata, you'll need to set `downloadFile=False` ([docs](https://python-docs.synapse.org/build/html/index.html#synapseclient.Synapse.get)).